### PR TITLE
Rework uwu (#78 issue)

### PR DESCRIPTION
**Pull request by aggyomfg**
Removed `/uwu` command
Added uwu\uvvu regex filter
Added message handler to check every text message

Used images (send it to bot directly or in group, so it can get file_id's):
https://i.redd.it/cqpuzj8avzh11.png
https://i.redd.it/gv0zhqyanmh11.jpg
https://pbs.twimg.com/profile_images/984487122517274625/pqXXdSIP.jpg

### CODE REVIEW

**Comment by aggyomfg** [comment]
BTW does someone know, file_id fixed in group or somehow else? Or i should change file_id to urls?

File: bot/filters.py [hunk 1]
```diff
@@ -11,4 +12,16 @@ def filter(self, message) -> bool:
         }
 
 
+class UwuFilter(BaseFilter):
+    """ Regexp check for UwU """
+    name = 'Filters.uwu'
+
+    def filter(self, message) -> bool:
+        if message.text:
+            return bool(re.search(r'u(w+|v+)u', message.text, re.IGNORECASE))
```
[the remark this answers is not in the document]
Comment (egregors) [reply bot/filters.py]: @C-Pro `(^|\s+)u(w+|v+)u($|\s+)` does not match UwU and UvvU

**Comment by Grawl** [comment]
just upload images here to save originals

cqpuzj8avzh11
gv0zhqyanmh11
pqXXdSIP

File: bot/filters.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Grawl) [reply bot/filters.py]: i think we need word boundries here, `\b`

`\bu[wv]+u\b` https://regex101.com/r/O28cUg/1

[the remark this answers is not in the document]
Comment (egregors) [reply bot/filters.py]: @Grawl looks like this is the best one

[the remark this answers is not in the document]
Comment (C-Pro) [reply bot/filters.py]: > @C-Pro `(^|\s+)u(w+|v+)u($|\s+)` does not match UwU and UvvU
With `re.IGNORECASE`?

[the remark this answers is not in the document]
Comment (Grawl) [reply bot/filters.py]: + we need case ignoring here

**Pull request merged**